beanstalk_worker/worker_start_build.py

```diff
@@ -143,17 +136,20 @@ def start_build(job_details):
             logs = run_command(log_command)
         except Exception as e:
             logger.log(level=logging.CRITICAL, msg=e.message)
-            logs = "Could not retrieve build logs"
+            logs = "Could not retrieve build logs."
+            logger.log(level=logging.CRITICAL, msg=logs)
         else:
             logger.log(level=logging.INFO,
                        msg="Writing build logs to NFS share..")
+        finally:
+            # Export logs on disk in either case
             export_build_logs(logs, build_logs_file)
 
         is_complete = run_command(status_command).find('Complete')
```

Comment:
What is `run_command` returning? `is_complete` variable makes me feel it should be a True/False scenario. But `< 0` check makes me think I'm wrong. Also, `run command` function has no comments on what it's doing! It's a function that's executing some command but unclear as to what it's returning.